src/docs/asciidoc/setting_up_clusters.adoc

```diff
@@ -35,7 +35,7 @@ Hazelcast supports EC2 auto-discovery. It is useful when you do not want to prov
 [[gcp-cloud-discovery]]
 ==== GCP Cloud Discovery
 
-Hazelcast supports discovering members in the https://cloud.google.com/compute/[GCP Compute Engine] environment. You can easily configure Hazelcast members discovery, WAN replication, and Hazelcast Client to work seamlessly on the native GCP VM Instances. Please see its https://github.com/hazelcast/hazelcast-gcp/blob/master/README.md[documentation] for information on configuring and using it.
+Hazelcast supports discovering members in the https://cloud.google.com/compute/[GCP Compute Engine] environment. You can easily configure Hazelcast members discovery, WAN replication, and Hazelcast Client to work seamlessly on the native GCP VM Instances. This discovery feature is provided as a Hazelcast plugin. Please see its https://github.com/hazelcast/hazelcast-gcp/blob/master/README.md[documentation] for information on configuring and using it.
```

Comment:
Is this not-being-a-plugin specific to GCP discovery (because it is included in the Hazelcast Plugins chapter)?

Reply:
Yeha, this part is related to the GCP Plugin. I added it just to be constistent with other plugins (like AWS) as recommended by Kyle [1]

[1] https://hazelcast.atlassian.net/wiki/spaces/PM/pages/892764790/Google+Cloud+Platform+Integration+Technical+Design+Document?focusedCommentId=1078558754